lang/cs/Org.Apache.REEF.Client.Tests/YarnREEFParamSerializerTests.cs

```diff
@@ -83,10 +83,11 @@ public void TestYarnREEFDotNetJobSerialization()
                 "\"jobSubmissionFolder\":\"{0}\"" +
                 "}}," +
                 "\"dfsJobSubmissionFolder\":\"{0}\"," +
+                "\"fileSystemUrl\":\"{1}\"," +
                 "\"jobSubmissionDirectoryPrefix\":\"{0}\"" +
                 "}}";
 
-            var expectedJson = string.Format(formatString, AnyString);
+            var expectedJson = string.Format(formatString, AnyString, "NULL");
```

Comment:
we can use a const value here, something like `FileSystemUrl.DefaultValue` (please see my comment below)